cpp/unreal_plugins/UrdfBot/Source/UrdfBot/UrdfBotPawn.cpp

```diff
@@ -0,0 +1,81 @@
+//
+// Copyright(c) 2022 Intel. Licensed under the MIT License <http://opensource.org/licenses/MIT>.
+//
+
+#include "UrdfBotPawn.h"
+
+#include <Camera/CameraComponent.h>
+#include <Components/InputComponent.h>
+
+#include "CoreUtils/Config.h"
+#include "CoreUtils/Unreal.h"
+#include "UrdfRobotComponent.h"
+#include "UrdfParser.h"
+
+AUrdfBotPawn::AUrdfBotPawn(const FObjectInitializer& object_initializer): APawn(object_initializer)
```

Comment:
More consistent white space with the rest of the code.
 
```
AUrdfBotPawn::AUrdfBotPawn(const FObjectInitializer& object_initializer) : APawn(object_initializer)
```

Reply:
fixed